pkg/stanza/fileconsumer/roller_other.go

```diff
@@ -30,15 +30,18 @@ OUTER:
 			if newReader.Fingerprint.StartsWith(oldReader.Fingerprint) {
 				continue OUTER
 			}
-			if oldReader.FileName == newReader.FileName {
-				// At this point, we know that the file has been rotated. However, we do not know
-				// if it was moved or truncated. If truncated, then both handles point to the same
-				// file, in which case we should only read from it using the new reader. We can use
-				// the validateFingerprint method to establish that the file has not been truncated.
-				if !oldReader.ValidateFingerprint() {
-					continue OUTER
-				}
+			if oldReader.FileName != newReader.FileName {
+				continue
 			}
+
+			// At this point, we know that the file has been rotated. However, we do not know
+			// if it was moved or truncated. If truncated, then both handles point to the same
+			// file, in which case we should only read from it using the new reader. We can use
+			// the validateFingerprint method to establish that the file has not been truncated.
```

Comment:
validateFingerprint -> ValidateFingerprint

Reply:
Good catch. The method name ended up changing because of another PR but I made sure to get it right this time.